_observing-your-data/alerting/dashboards-alerting.md

```diff
@@ -0,0 +1,92 @@
+
+## General requirements for alerting visualizations
+
+Alerting visualizations are displayed as time-series charts that give you a snapshot of the alert, alert status, last updated time, and reason for the alert. You can show up to 10 metrics on your chart, and each series can be shown as a line in the chart.
+
+Keep in mind the following requirements when setting up or creating alerting visualizations:
+
+- Must be a [Vizlib line chart](https://community.vizlib.com/support/solutions/articles/35000107262-vizlib-line-chart-introduction)
+- Must contain at least Y-axis metric aggregation
+- Must not have non-Y-axis metric aggregation types
+- Must use date histogram aggregation type for the X-axis bucket
+- Must have X-axis on the bottom
+- Must define one X-axis aggregation bucket
+- Must have a valid time-based X-axis
+
+## Creating alerting monitors
+
+By default, when you begin to create the alert monitor workflow using the Dashboard interface, you are presented with a menu-driven interface. This interface provides a range of options that are displayed in full screen, in pop-ups, in pull-downs, or dropdowns. They allow you to define the metrics that can be monitored, set thresholds, customize triggers that automate workflows, and generate actions when conditions are met. Currently, you can create [per query monitors]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/monitors/#monitor-types) only.
+
+#### To create an alerting monitor 
+
+1. Choose **Dashboard** from the OpenSearch Dashboards main menu.
+2. From the **Dashboards** window, select **Create** and then choose **Dashboard**.
+3. Select **Add an existing**, then select the appropriate alerting visualization from the **Add panels** list. The visualization is added to the dashboard.
+4. From the visualization panel, choose the ellipsis icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/ellipsis-icon.png" class="inline-icon" alt="ellipsis icon"/>{:/}). 
+5. From the **Options** menu, select **Add alerting monitor**.
+6. Input information for **Monitor details** and **Triggers**.
+7. Choose **Create monitor**. The monitor is added to the visualization.  
+
+An example of the create monitor steps is shown in the following screenshot.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/create-monitor-menu.png" alt="Create monitor interface" width="400" height="400">
+
+## Associating monitors
+
+You can associate certain monitor types with a visualization using the Dashboard interface instead of the plugin page, giving you a single interface to add, view, and edit monitor data.
+
+#### To associate a monitor
```

Comment:
This heading reads a bit oddly because it's used as a procedure heading above but as a section heading here (no sentence follows it in the preceding instance).